.github/workflows/platform_ci_workflow.yml

```diff
@@ -11,19 +11,14 @@ on:
         type: string
         description: "type of model to execute"
         required: true
-    secrets:
-      subscription_id:
-        description: "the subscription ID hosting the AML instance."
-        required: true
-      resource_group_name:
-        description: "the resource group name hosting the AML instance."
-        required: true
-      workspace_name:
-        description: "the workspace name hosting the AML instance."
-        required: true
-      azure_credentials:
-        description: "the credentials to use to authenticate with Azure."
-        required: true
+env:
```

Comment:
I wonder if the variables are going to be propagated from the parent pipeline?

Reply:
They weren't.  I tried that and in this case it didn't work